orion-ui/src/App.vue

```diff
@@ -12,13 +12,7 @@
     </template>
     <ContextSidebar v-if="showMenu" class="app__sidebar" @click="close" />
     <suspense>
-      <router-view class="app__router-view">
-        <template #default="{ Component }">
-          <transition name="app__router-view-fade" mode="out-in">
-            <component :is="Component" />
-          </transition>
-        </template>
-      </router-view>
+      <RouterView />
```

Comment:
I like this change but I'd use a different name. It's too easy to assume that's the same router view component provided by vue-router

Reply:
Open to alternatives, any suggestions?

Reply:
AppRouterView, RootRouterView, ProvideRouterView?

Reply:
Went with `AppRouterView`